### ISSUE TITLE
[Feature Request] Add launch browser feature when `docfx serve` command is executed

### ISSUE THREAD

**Issue by filzrev** [dotnet/docfx#8927]
**Is your feature request related to a problem? Please describe.**

There is [dotnet-serve](https://github.com/natemcmaster/dotnet-serve) tools that support this feature.
It is useful when using docfx with custom MSBuild project and IDE integration.

**Describe the solution you'd like**
Add `--open-browser` parameter to `docfx serve` and `docfx build --serve` commands.
When this parameter specified. docfx launch new browser instance with default URL (http://localhost:{port}).

Additionally.  when relative path of markdown file is passed as argument.
It'll be resolved to corresponding URL by using `manifest.json`.  
This feature is useful when `docfx serve` command is execute via VSCode tasks.  
(VSCode support passing current file as `${relativeFile}` parameter)

**Additional context**
If above feature request is acceptable. I'll create PR.

**Comment by yufeih** [comment]
Yes! Sounds like a great addition!

**Comment by filzrev** [comment]
Features are Implemented by #8934
Following options are added for `docfx build --serve` and `docfx serve` commands.

```
--open-browser  Open a web browser when the hosted website starts
--openfile      Open a file in a web browser when the hosted website starts.
```
